adapters/google/google.go

```diff
@@ -0,0 +1,229 @@
+package google
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
```

Comment:
"io/ioutil" is deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code.

Reply:
resolved